processing/src/main/java/io/druid/segment/data/GenericIndexed.java

```diff
@@ -370,11 +369,10 @@ public void writeTo(WritableByteChannel channel, FileSmoosher smoosher) throws I
 
   private T copyBufferAndGet(ByteBuffer valueBuffer, int startOffset, int endOffset)
```

Comment:
The logic of this method should be the same as `bufferedIndexedGet()`, but now it could pass -1 to `strategy.fromByteBuffer()`.

Please review this aspect again across `GenericIndexed` and `GenericIndexedWriter`.

Reply:
done.